queries/just/indents.scm

```diff
@@ -0,0 +1,8 @@
+; Source: https://github.com/IndianBoy42/tree-sitter-just/blob/main/queries/just/indents.scm
+[
+  (recipe)
+  (string)
+  (external_command)
+] @indent.begin
```

Comment:
String with indent begin seems off.
I'd say just let it be

Reply:
This is because Just strips common leading whitespace for multiline strings, e..g.

```just
# becomes `foo\nbar\n`
x := '''
  foo
  bar
'''
```

ls is nice that it continues indent while you're typing a string, but indeed a bit weird that it keeps the indent after the string. Not sure if there is a better way to do this.